tests/rbd/test_concurrent_write_pwl_cache.py

```diff
@@ -0,0 +1,243 @@
+"""RBD Persistent write back cache, Test concurrent writes to same image
+
+Test Case Covered:
+CEPH-83574720 -
+Concurrent writes to check exclusive lock feature with persistent cache enables
+via same or different IO tools in SSD mode
+
+Steps :
+1) Setup persistent write cache for an image
+2) Write data to the same image concurrently using two or more long running fio jobs
+on different mount points.
+3) Verify that data written is present in both mount points and persistent cache file
+is generated while data is being written
+
+Environment and limitations:
+ - The cluster should have 5 nodes + 1 SSD cache node
+ - cluster/global-config-file: config/quincy/upi/octo-5-node-env.yaml
+ - Should be Bare-metal.
+
+Support
+- Configure cluster with PWL Cache.
+- Only replicated pool supported, No EC pools.
+"""
+import json
+
+from ceph.parallel import parallel
+from ceph.utils import get_node_by_id
+from tests.rbd.krbd_io_handler import krbd_io_handler
+from tests.rbd.rbd_peristent_writeback_cache import (
+    PersistentWriteAheadLog,
+    PWLException,
+    get_entity_level,
+)
+from tests.rbd.rbd_utils import initial_rbd_config
+from utility.log import Log
+from utility.utils import run_fio
+
+log = Log(__name__)
+
+
+def unmount(rbd, mount_point):
```

Comment:
clean up code is already there here https://github.com/red-hat-storage/cephci/blob/6692c56f9e02a83a84527e24b922e8a02e45d296/tests/rbd/rbd_peristent_writeback_cache.py#L33
can we reuse the same.

Reply:
That cleanup does only persistent cache settings cleanup. It doesn't unmount. So it can't be used.